app/controllers/pages_controller.rb

```diff
@@ -21,13 +21,20 @@ def collect
   end
 
   def autocomplete
-    render json: Page.search(params[:query], {
-      fields: ["scientific_name^5"],
-      match: :word_start,
-      # limit: 10,
-      load: false,
-      misspellings: false
-    })
+    render json:
+      JSON.parse(Page.search((params[:query]), 
+        {
+          fields: ["name_string"], 
+          match: :word_start, 
+          load: false, 
+          misspellings: false}
```

Comment:
what does "misspellings" key mean?

Reply:
If set to false, it means that the results will not include matches that are some letters off the query.
E.g. if I search for "golden-brested", results containing "golden-breasted" will not be included because it's been specified that one wouldn't search with a misspelled query.